### PR TITLE
Added GET /order/{id}

### PR DESCRIPTION
**Pull request by captaincoordinates**
Added GET /order/{id} docs

### CODE REVIEW

File: order/README.md [hunk 1]
```diff
@@ -6,28 +6,35 @@ Ordering with loosely defined order values will give the provider more freedom t
...
 | product_id         | string                                                                     | **REQUIRED.** Product identifier. The ID should be unique and is a reference to the constraints which can be used in the constraints field. |
 | geometry   | [GeoJSON Geometry Object](https://tools.ietf.org/html/rfc7946#section-3.1) | **REQUIRED.** Defines the full footprint of the asset represented by this item, formatted according to [RFC 7946, section 3.1](https://tools.ietf.org/html/rfc7946#section-3.1). The footprint should be the default GeoJSON geometry, though additional geometries can be included. Coordinates are specified in Longitude/Latitude or Longitude/Latitude/Elevation based on [WGS 84](http://www.opengis.net/def/crs/OGC/1.3/CRS84). |
 | filter | CQL2 JSON | A set of additional constraints in [CQL2 JSON](https://docs.ogc.org/DRAFTS/21-065.html) based on the constraints exposed in the product. |
 
-## Response
+## Create Order Response
+See [Order Object](#order-object).
+
+# GET /orders
```
Comment (m-mohr) [inline order/README.md:20]: You can't have multiple top-level headers.

**Review by m-mohr — CHANGES_REQUESTED** [review]
Fix markdown issues and update the top-level readme references

File: order/README.md (hunk 1 above)
Comment (captaincoordinates) [reply order/README.md:20]: that's a markdown rule or a STAC/T rule?

Comment (m-mohr) [reply order/README.md]: I think that's a Markdown rule, at least my linter complains ;)

Comment (captaincoordinates) [reply order/README.md]: Do you have more linter rules disabled locally than are in `.markdownlint.json`? Using the linter there seem to be a number of remaining problems.

Comment (m-mohr) [reply order/README.md]: No, my linter doesn't seem to use this config file at all.

**Review by CloudNiner — APPROVED** [review]

**Review by m-mohr — APPROVED** [review]

**Pull request merged**